docs/source/ui/pages/run_management.md

```diff
@@ -189,3 +199,155 @@ Use the `Settings` tab to delete or archive the single run
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Unarchive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/unarchive_card.png">
+
+## Experiment page
+
+The experiment page provides overall information about a single experiment and helps to explore the runs attached to the experiment. 
+Here are the tabs available on the experiment page:
+
+- [Overview](#id21)
+- [Runs](#id24)
+- [Notes](#id25)
+- [Settings](#id26)
+
+### Overview
+
+The Overview tab provides an overview of the runs attached to the experiment. 
+It displays general information such as their status, progress, and other relevant metadata.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page overview tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-overview-tab.png">
+
+#### Statistics
+
+On this page, you can view the total number of runs, as well as the number of runs that are `archived`, `active`, and `finished`.
+You can click on the corresponding cards to view more information about these runs, and you can also use these cards to navigate to the [Runs explorer](../pages/run_management.html#runs-explorer) page with a specific query.
+The statistics bar, below the `status` cards, displays the distribution of runs by status.
+
+#### Contributions calendar
+
+The contributions calendar represents the intensity of executed runs during the previous year. More on this read [here](../pages/home_page.html#id8).
+
+#### Activity feed
+
+This section represents the activity feed of your contributions related to the experiment. More on this read [here](../pages/home_page.html#id9).
+
+### Runs
+
+The Runs tab shows the list of runs attached to the experiment, and it also gives the ability to compare them using the available explorers.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page runs tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-runs-tab.png"> 
+
+### Notes
+
+The Notes tab is for attaching simple text or rich Markdown as a note to the experiment.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page notes tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-notes-tab.png">
+
+### Settings
+
+The Settings tab is for editing the experiment's name and description.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page settings tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-settings-tab.png">
+
+## Tags page
+
+### Overview
+
+The Tags functionality is intended to mark runs. A tag can be attached to runs to segment them and then quickly find them.
+
+### Create tag
+
+There are two options for creating a tag:
+
+##### First option
```

Comment:
I would not just write first option. I would be more specific.
Something like `Option 1: From Tags page